docs/_documentations/remote-removing.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: docs
+title: Removing Codewind remote
+description: Removing Codewind remote
+keywords: users, projects, Kubernetes, LDAP, user management, access management, login, deployment, pod, security, securing Cloud connection
+duration: 5 minutes
+permalink: remoteremoving
+type: document
+parent: installoncloud
+order: 1
+---
+
+## Removing Codewind remote
+
+To remove Codewind remote, enter the following `cwctl` command: `cwctl remove remote --namespace {mynamespace} --workspace {workspaceID}`
```

Comment:
Probably should explain how to get the workspace ID value for the --workspace argument.  Can do via kubectl I supposed, but perhaps cwctl should be enhanced to easily get the workspace ID after a remote install?